edb/edgeql/ast.py

```diff
@@ -623,7 +623,7 @@ class SetSpecialField(BaseSetField):
 class ObjectDDL(DDLCommand, CompositeDDL):
     __abstract_node__ = True
     name: ObjectRef
-    commands: typing.Sequence[DDLCommand]
+    commands: typing.List[DDLCommand]
```

Comment:
I am not sure this change is right. But it looks reasonable, since we use the method "append" on commands property, so assuming it looks like a `List`?

Reply:
Yes, `List` is correct.